### PR TITLE
feat(BidSnapshot): L3-4690 changed prop

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-4690](https://phillipsauctions.atlassian.net/browse/L3-4690)

**Screenshots**
| Before | After |
| ------ | ----- |
| image | image |

**Summary**

Renamed component's props to align with business logic

**Change List (describe the changes made to the files)**

- changed all files with `auctionStatus` to `lotStatus`

**Acceptance Test (how to verify the PR)**

- verify that we don't have `auctionStatus` in repo
- verify that `BidSnapshot` status works as it worked

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-4690]: https://phillipsauctions.atlassian.net/browse/L3-4690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/patterns/SaleHeaderBanner/SaleHeaderBanner.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Meta } from '@storybook/react';
 import SaleHeaderBanner, { SaleHeaderBannerProps } from './SaleHeaderBanner';
-import { AuctionStatus } from '../../types/commonTypes';
+import { LotAuctionStatus } from '../../types/commonTypes';
```
Comment (davidicus) [inline src/patterns/SaleHeaderBanner/SaleHeaderBanner.stories.tsx:3]: It does not make sense to change the name of this enum. SaleHeaderBanner logic is dependent on the state of the auction, not the individual lot.

File: src/patterns/SaleHeaderBanner/SaleHeaderBanner.test.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import { describe, it, expect } from 'vitest';
 import SaleHeaderBanner, { SaleHeaderBannerProps } from './SaleHeaderBanner';
-import { AuctionStatus } from '../../types/commonTypes';
+import { LotAuctionStatus } from '../../types/commonTypes';
```
Comment (davidicus) [inline src/patterns/SaleHeaderBanner/SaleHeaderBanner.test.tsx:4]: Same comment about status. SaleHeaderBanner is concerned with the status of the auction and not lot

File: src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { ComponentProps, forwardRef } from 'react';
 import { getCommonProps } from '../../utils';
 import classnames from 'classnames';
 import { SeldonImage } from '../../components/SeldonImage';
-import { AuctionStatus } from '../../types/commonTypes';
+import { LotAuctionStatus } from '../../types/commonTypes';
```
Comment (davidicus) [inline src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx:5]: Same comment about auction status

File: src/types/commonTypes.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ export enum SupportedLanguages {
   zh = 'zh',
 }
 
-export enum AuctionStatus {
+export enum LotAuctionStatus {
```
Comment (davidicus) [inline src/types/commonTypes.ts:6]: We will probably need two separate enums here. I assumed the lot status check would have been done in the remix project. If we are adding an additional enum here we need to still support AuctionStatus.

**Review by davidicus — DISMISSED** [review]
Just the comment about the Sale and lot status enum.

**Review by scottdickerson — APPROVED** [review]
LGTM

**Pull request merged**